aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -47,6 +48,34 @@
                 { preventDefault: function () {} })
             ).toEqual(false);
         });
+
+        it('Cl.events._handler returns undefined if direction is "next" and month is 7', function () {
+            expect(Cl.events._handler.call(
+                $('.js-trigger')[1],
+                { preventDefault: function () {} })
```

Comment:
I'd suggest following the DRY principle, for example you could write:

```
var noop = function () {};

{ preventDefault: noop })
```

or go even further:

```
var prevent = { preventDefault: function () {} }
expect(Cl.events._handler.call(
    $('.js-trigger')[2], prevent).toEqual(undefined);
```